docs/diagrams/PlantUML File/StorageUpdatePropertySequenceDiagram.puml

```diff
@@ -0,0 +1,50 @@
+@startuml
+'https://plantuml.com/sequence-diagram
+
+hide footbox
+
+Participant "storage : Storage" as Storage #F4F4B6
+Participant "propertyFile : FileWriter" as FileWriter #F4F4B6
+Participant "propertyList : PropertyList" as PropertyList #F4F4B6
+Participant "property : Property" as Property #F4F4B6
+
+->Storage:updateProperty()
+activate Storage #CDEFF9
+
+create FileWriter
+Storage -> FileWriter:FileWriter(PROPERTY_PATH)
+activate FileWriter #CDEFF9
+return
+
+Storage -> PropertyList:getPropertyList()
+activate PropertyList #CDEFF9
+return propertyList
+
+    loop for each property in propertyList
+        Storage -> Property:getLandlordName()
```

Comment:
In your code, there are multiple `propertyLists.get(i)` calls which are not reflected in your sequence diagram.
Maybe you can refactor your code to make a single call instead